semgrep-core/src/engine/Specialize_formula.ml

```diff
@@ -31,11 +40,22 @@ let selector_equal s1 s2 = s1.mvar = s2.mvar
 (* Converter *)
 (*****************************************************************************)
 
+(* return list of "positive" x list of Not x list of Conds *)
```

Comment:
I think this comment got garbled? Definitely should have something that specifies this is (positive, negative, conditions)